[InstagramBridge] No Media in Content Option

nvg123 opened the pull request:
Hi,

Here is the suggestion to add an option that removes Media from Content. 
Sometimes (e.g. in JSON) you don't need Media in Content, because it already listed in Attachment\Enclosure. This pull reqest adds an option for that.

nvg123 committed:
Update InstagramBridge.php

nvg123 committed:
Option not to include Media in Content

Sometimes you don't need Media in Content, because it already listed in Attachment\Enclosure. This commit adds an option for that

nvg123 committed:
Fixing broken ▶ sign

nvg123 committed:
And another ▶ fix :)

nvg123 committed:
Fixing The Travis CI build detected failures

  56 | ERROR | [x] Whitespace found at end of line
 147 | ERROR | [x] Whitespace found at end of line
 151 | ERROR | [x] Whitespace found at end of line
 185 | ERROR | [x] Whitespace found at end of line
 193 | ERROR | [x] Whitespace found at end of line
 207 | ERROR | [x] Whitespace found at end of line
 211 | ERROR | [x] Whitespace found at end of line

nvg123 committed:
Fixing The Travis CI build detected failures

 193 | ERROR | [x] Whitespace found at end of line

somini commented:
I get warnings on using this. Here's a fix: https://github.com/somini/rss-bridge/commit/0a882aa945ea165b6ceeb925e51dfb58503493e4

You can apply this with:

```
git pull "https://github.com/somini/rss-bridge.git" pull/1399
```

nvg123 committed:
Adding fix from somini

somini@0a882aa
https://github.com/somini/rss-bridge/commit/0a882aa945ea165b6ceeb925e51dfb58503493e4
